includes/transformer/class-comment.php

```diff
@@ -63,8 +63,9 @@ public function to_object() {
 		$comment = $this->wp_comment;
 		$object = new Base_Object();
 
-		$object->set_id( $this->get_id() );
+		$object->set_id( $this->get_id( $comment ) );
 		$object->set_url( \get_comment_link( $comment->ID ) );
+		$object->set_context( $this->get_context() );
```

Comment:
same as above... we have the comment as object attribute... it feels a bit weird to use the getter to fill the setter...